ambry-admin/src/test/java/com.github.ambry.admin/AdminBlobStorageServiceTest.java

```diff
@@ -389,7 +392,8 @@ public void getReplicasTest() throws Exception {
     List<? extends PartitionId> partitionIds = CLUSTER_MAP.getWritablePartitionIds();
     for (PartitionId partitionId : partitionIds) {
       String originalReplicaStr = partitionId.getReplicaIds().toString().replace(", ", ",");
-      BlobId blobId = new BlobId(partitionId);
+      BlobId blobId = new BlobId(BlobId.DEFAULT_FLAG, ClusterMapUtils.LEGACY_DATACENTER_ID, Account.LEGACY_ACCOUNT_ID,
```

Comment:
Good catch, thanks! As discussed with Siva before, they were temporarily set in `BlobProperties`. Will remove them.